src/custom/state/application/hooks.ts

```diff
@@ -42,3 +48,31 @@ export function useAddPopup(): (content: PopupContent, key?: string, removeAfter
     [dispatch]
   )
 }
+
+// Allow show popup
+// actions to use followPendingTxPopup atom
+export const useFollowPendingTxPopup = () => {
```

Comment:
To be honest, I don't think this hook is needed at all.
The atoms are very simple and this is only re-exposing the atom read/write methods.

Reply:
I see, my intention was also to have actions in a single import point (and not to use _useAtom_/_useUpdate_/_useValue_ on Components).  I'll use a simple function to accomplish the same thing.